src/stdlib/String.ts

```diff
@@ -61,9 +60,67 @@ export const Chr = new Callable(
  */
 export const Left = new Callable(
     {
-        name: "Left",
-        args: [{ name: "s", type: ValueKind.String }, { name: "n", type: ValueKind.Int32 }],
-        returns: ValueKind.String,
+      name: 'Left',
+      args: [{name: 's', type: ValueKind.String}, {name: 'n', type: ValueKind.Int32}],
+      returns: ValueKind.String
     },
     (interpreter: Interpreter, s: BrsString, n: Int32) => new BrsString(s.value.substr(0, n.getValue()))
 );
+
+/**
+ * Returns the last n characters in a string
+ */
+export const Right = new Callable(
+    {
+      name: 'Right',
+      args: [{name: 's', type: ValueKind.String}, {name: 'n', type: ValueKind.Int32}],
+      returns: ValueKind.String
+    },
+    (interpreter: Interpreter, s: BrsString, n: Int32) => {
+        let end = s.value.length - 1;
+        let start = end - (n.getValue() - 1);
+
+        if (n.getValue() <= 0) return new BrsString("");
```

Comment:
Looks like the [BrightScript docs](https://sdkdocs.roku.com/display/sdkdoc/Global+String+Functions#GlobalStringFunctions-Right(sasString,nasInteger)asString) don't specify the behavior here &ndash; is that how it works in the reference implementation on a Roku?  I can't remember at the moment :sweat_smile:

Reply:
I will check on my Roku, but I believe this is correct. I will add this to the e2e test as well and run in both places (brs and my Roku).